test/capitalize.spec.js

```diff
@@ -1,13 +1,13 @@
 'use strict';
 
-var capitalizeFirst = require('../').capitalizeFirst;
+var capitalize = require('../').capitalize;
 var tape = require('tape');
 var Handlebars = require('handlebars');
 
-Handlebars.registerHelper(capitalizeFirst.name, capitalizeFirst);
+Handlebars.registerHelper(capitalize.name, capitalize);
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/name

But now that you mention it, it's kind of silly to use `name` on a reference that happens to be using the same value as its own identifier. It would make more sense if we were doing this:

```js
const helper = require('path/to/helper');

Handlebars.registerHelper(helper.name, helper);
```